tests/test_profile_page.py

```diff
@@ -0,0 +1,65 @@
+"""Tests related to profile page"""
+import logging
+
+import pytest
+
+from constants import header
+
+
+class TestProfilePage:
+    """Stores tests for profile page base functionality"""
+
+    log = logging.getLogger("[TestProfilePage]")
+
+    @pytest.fixture()
+    def navigate_to_profile_page_via_sign_in(self, start_page, random_login):
+        """
+        - Pre-conditions:
+            - Sign In as a user
+        - Steps:
+            - Click on "My Profile" button
+            - Navigate to profile page
+            - Verify profile username
+        """
+        # Sign In as a user
+        hello_page = start_page.sign_in(random_login)
+
+        # Verify login success
+        hello_page.verify_sign_up_message(username=random_login.username_value)
+
+        # Click on "My Profile" button
+        header.my_profile_button()
+
+        # Navigate to profile page
+        profile_page = hello_page.navigate_to_profile_page()
+        return profile_page
+
+        # Verify profile username
+        profile_page.verify_profile_page_message(username=random_login.username_value)
```

Comment:
Why do you do verifications under the fixture ?

Reply:
![image](https://user-images.githubusercontent.com/47011239/215155213-1e184cec-d5ef-4350-9cbb-6b1b19d25d46.png)
I check the username message on the page https://qa-complexapp.onrender.com/profile/